lib/pago.rb

```diff
@@ -0,0 +1,31 @@
+require 'ostruct'
+
+class Pago
+  class << self
+    ACCEPTED_PAYMENTS = [
+      :check,
+      :credit_card,
+      :po,
+    ]
+
+    def make_payment(order_id:, payment_method:, payment_details:)
+      raise "Unknown payment_method #{payment_method}" unless ACCEPTED_PAYMENTS.include?(payment_method)
```

Comment:
You might know better than I would as far as the convention for Ruby goes, but it seems like it'd be better to raise an error class, rather than just raise with a message. I imagine this would just raise `StandardError`?

Maybe create a class with a message for it and raise that if it fails your check?